gradle/libs.versions.toml

```diff
@@ -1,6 +1,7 @@
 [versions]
 accompanist = "0.30.1"
 android-gradlePlugin = "8.2.2"
+colorPicker = "1.0.0"
```

Comment:
can we keep the case consistent with the library declaration, either `colorPicker` or `colorpicker`

Reply:
Alright, I removed this repo actually not required anymore